src/app_charts/base/cloud/registry-crd.yaml

```diff
@@ -54,3 +54,32 @@ spec:
               type: string
             project:
               type: string
+        status:
+          type: object
+          properties:
+            cloud:
+              type: object
+            robot:
+              type: object
+              properties:
```

Comment:
why robot/cloud split here?

Reply:
This is just reflecting the structure from annotations https://github.com/googlecloudrobotics/core/blob/dddd7e00345351e7906c43ac86a08b41adecd880/src/app_charts/base/cloud/registry-crd.yaml#L34-L36
Afaik originally it was used to be able to send missions from cloud cluster to the robot clusters.

Reply:
Ahh, right you're just adding the validation for the existing use.